AUTHORS

```diff
@@ -0,0 +1,16 @@
+Corporate Contributors
+======================
+
+Copyright (c) 2015-2017 IBM
+Copyright (c) 2016-2017 Docker, Inc.
+
+Individual Contributors
+=======================
+
+Dan Williams
+Hannes Mehnert
+Ian Campbell
+Madhuri Yechuri
+Martin Lucina
+Ricardo Koller
+Waldir Pimenta
```

Comment:
Thanks @avsm, this is a good idea. I've addressed it and re-pushed this as 07ab01f.